test-storybooks/portable-stories-kitchen-sink/react/package.json

```diff
@@ -108,7 +109,9 @@
     "@typescript-eslint/eslint-plugin": "^6.21.0",
     "@typescript-eslint/parser": "^6.21.0",
     "@vitejs/plugin-react": "^4.2.1",
-    "@vitest/browser": "^2.1.3",
+    "@vitest/browser": "^2.1.6",
+    "@vitest/coverage-v8": "^2.1.6",
+    "@vitest/ui": "2.1.8",
```

Comment:
you might not want to have mismatching versions here, this should be either ^2.1.6 or all other vitest deps should be 2.1.8

Reply:
right, Vitest added the 2.1.8 automatically here.